### ISSUE TITLE
Allow attribution flag to resize with font-size

### ISSUE THREAD

**Issue by Malvoz** [Leaflet/Leaflet#8151]
The 🇺🇦 flag does currently not scale with font-size changes unlike the rest of the attribution.

`width: 1em` and `height: auto` should be sufficient in allowing the flag to resize with `font-size` changes.

Though `em` is not supported in IE 6-9, so perhaps something like the following is required:

```diff
var ukrainianFlag = '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="12" height="8"
+ viewBox="0 0 12 8
">
```

```diff
.leaflet-control-attribution svg {
	display: inline !important;
+	width: 1em;
+	height: auto;
	}
```

The CSS takes precedence over the presentational attributes, allowing the flag to resize in relation to font-size in browsers that support `em` units.

**Comment by sumitsaurabh927** [comment]
Hey @Malvoz I'd like to work on this. Can you please assign me?

**Comment by sumitsaurabh927** [comment]
I'll get started in this! Thanks for assigning me!

**Comment by Malvoz** [comment]
IMO, we may as well take the opportunity and set a dedicated CSS class per https://github.com/Leaflet/Leaflet/pull/8119#discussion_r842896686.

For example:

[Control.Attribution.js#L9](https://github.com/Leaflet/Leaflet/blob/da386bec18d2e103acb207e97cf332bd388bb575/src/control/Control.Attribution.js#L9):

```diff
var ukrainianFlag = '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="12" height="8"
+ viewBox="0 0 12 8" class="leaflet-attribution-flag"
">
```

[leaflet.css#L426-L429](https://github.com/Leaflet/Leaflet/blob/da386bec18d2e103acb207e97cf332bd388bb575/dist/leaflet.css#L426-L429):

```diff
- .leaflet-control-attribution svg {
+ .leaflet-attribution-flag {
	display: inline !important;
+	width: 1em;
+	height: auto;
	}
```

**Comment by sumitsaurabh927** [comment]
Sure, sounds great! Will set a dedicated class for it!

**Comment by sumitsaurabh927** [comment]
I'm having a little problem running the server to see things in action. I went through the conrtibutor's guide and ran rollup but when I issue 'npm start' the server doesn't start. How do I see the deployed version where I can see the flag?

**Comment by Falke-Design** [comment]
run `npm run watch` and then open `/debug/map/map-mobile.html`

**Comment by sumitsaurabh927** [comment]
I've made the changes and tested and it works as expected!

**Comment by sumitsaurabh927** [comment]
But I made the changes in the main branch of my fork, shall I go ahead and raise a PR?

**Comment by Falke-Design** [comment]
Yes simply make a PR

**Comment by sumitsaurabh927** [comment]
Done! I've raised a PR. Please review it and merge if things are okay.
